frontend/src/pages/modelServing/customServingRuntimes/utils.ts

```diff
@@ -41,29 +41,44 @@ export const getServingRuntimeDisplayNameFromTemplate = (template: TemplateKind)
 export const getServingRuntimeNameFromTemplate = (template: TemplateKind) =>
   template.objects[0].metadata.name;
 
-export const isServingRuntimeKind = (obj: K8sResourceCommon): obj is ServingRuntimeKind =>
-  obj.kind === 'ServingRuntime' &&
-  obj.spec?.containers !== undefined &&
-  obj.spec?.supportedModelFormats !== undefined;
+const createServingRuntimeCustomError = (name: string, message: string) => {
+  const error = new Error(message);
+  error.name = name;
+  return error;
+};
+
+export const isServingRuntimeKind = (obj: K8sResourceCommon): obj is ServingRuntimeKind => {
+  if (obj.kind !== 'ServingRuntime') {
+    throw createServingRuntimeCustomError('Invalid parameter', 'kind: must be ServingRuntime.');
+  }
+  if (!obj.spec?.containers) {
+    throw createServingRuntimeCustomError('Missing parameter', 'spec.container: is required.');
```

Comment:
Is it `spec.containers` instead of `spec.container`?

Reply:
Yes you are right, sorry for the typo, will update soon.